visualization/app/codeCharta/ui/codeMap/codeMap.render.service.ts

```diff
@@ -50,7 +52,7 @@ export class CodeMapRenderService {
 		const appSettings = this.storeService.getState().appSettings
 		const showLabelNodeName = appSettings.showMetricLabelNodeName
 		const showLabelNodeMetric = appSettings.showMetricLabelNameValue
-		const highestNode = this.getHighestNode(sortedNodes)
+		this.highestNode = this.getHighestNode(sortedNodes)
```

Comment:
It is used to store a local copy for a highest node value that would otherwise not be availible to temporary labels. I decided to not move it ot the redux store because it is a local property and only concerns one service.